src/components/CardSchemes/CardSchemes.spec.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { keys } from 'lodash';
+
+import CardSchemes from './CardSchemes';
+// eslint-disable-next-line max-len
+import schemeMap from '../CreditCardDetails/components/scheme-icons/card-scheme-map';
+
+const iconSizes = [
+  CardSchemes.BYTE,
+  CardSchemes.KILO,
+  CardSchemes.MEGA,
+  CardSchemes.GIGA
+];
+
+describe('CardSchemes', () => {
+  /**
+   * Style tests.
+   */
+  it('should render with default styles', () => {
```

Comment:
I think I would like to see a test that checks that explicitly checks if the schemes have been rendered. You can do that by checking the scheme IDs, I would assume?

Reply:
The test was supposed to go here, to check that `CardSchemes` renders all the icons it receives as `schemeIds`.